Makefile

```diff
@@ -92,7 +92,8 @@ ifeq ($(NON_MATCHING),1)
 endif
 
 CPPFLAGS += -fno-dollars-in-identifiers -P
-LDFLAGS  := --no-check-sections --accept-unknown-input-arch --emit-relocs
+# TODO: Remove muldefs when all of libultra matches
+LDFLAGS  := --no-check-sections --accept-unknown-input-arch --emit-relocs -z muldefs
```

Comment:
What does `muldefs` do?

Reply:
Multiple definition warnings. Since I'm linking against the whole archive but also not using the whole archive yet, I need to silence multiple definition warnings to fully link.

Reply:
Oh, so you are both linking to individual files of the libultra archive and _also_ linking against the full archive? this sounds kinda bad, maybe there's something else misconfigured?

Reply:
Only while I can't match version H, since I'm mixing and matching which files from libultra I take from the archive. In my branch with a matching ultralib I remove this all together.

Reply:
Ohh, I see. I guess that makes a bit more of sense. Kinda annoying